### PR TITLE
Airgap

### PR DESCRIPTION
**Pull request by hdonnay**

### CODE REVIEW

File: Documentation/operation.md [hunk 1]
```diff
@@ -104,3 +104,127 @@ auth:
...
+### Configuration
+
+#### Airgap
+
+```yaml
+indexer:
+  airgap: true
+```
+
+#### Specific Scanners
```
Comment (ldelossa) [inline Documentation/operation.md:216]: what is this block for?

File: config/config.go [hunk 2]
```diff
@@ -55,6 +58,51 @@ type Config struct {
 	Auth     Auth     `yaml:"auth" json:"auth"`
 	Trace    Trace    `yaml:"trace" json:"trace"`
 	Metrics  Metrics  `yaml:"metrics" json:"metrics"`
+	Updaters Updaters `yaml:"updaters" json:"updaters"`
+}
+
+// Updaters configures updater behavior.
+type Updaters struct {
```
Comment (ldelossa) [inline config/config.go:65]: do we need to update the configs used in our repos now?

There is a sample config at the root of the repo,
./local-dev/clair/config.yaml is used in local dev.

**Review by ldelossa — CHANGES_REQUESTED** [review]

**Comment by ldelossa** [comment]
only blocking on whether sample and local dev configs need updating now, if not looks good to me.

File: Documentation/operation.md (hunk 1 above)
Comment (hdonnay) [reply Documentation/operation.md:216]: To have some documentation of how configuring scanners works.

I didn't sit down to write prose to flesh out these headings, though.

**Comment by hdonnay** [comment]
Will double-check that and ping you

File: Documentation/operation.md (hunk 1 above)
Comment (ldelossa) [reply Documentation/operation.md:216]: didn't realize config for scanners was being implemented. 

do we use that yet?

**Review by ldelossa — APPROVED** [review]

File: Documentation/operation.md (hunk 1 above)
Comment (hdonnay) [reply Documentation/operation.md:216]: It's wired up, now. In theory rhel uses it, but it's not end-to-end tested.

**Pull request merged**